gladier/tools/builtins/action.py

```diff
@@ -38,13 +44,21 @@ class ActionState(
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.exception_handlers: t.Dict[ActionExceptionName, GladierBaseState] = {}
+        self.exception_handlers: t.Dict[ActionExceptionName, AWSBaseState] = {}
 
     def set_exception_handler(
         self,
         exception_names: t.Union[ActionExceptionName, t.List[ActionExceptionName]],
-        exception_handler: GladierBaseState,
+        exception_handler: AWSBaseState,
     ):
+        """
```

Comment:
This doc string is correct on this. I guess this design for exception handling wasn't discussed much. It might be worth double checking that we're in sync on how this is handled.

Reply:
I haven't actually done that much with exception handlers in flows, opting instead to have the flow fail for unexpected behavior. I guess another way to say that is I haven't had unexpected behavior that I've wanted to treat differently than a fail state. I could try playing with it a little, but this seemed like a nice way to do it.

Reply:
I guess we stick with it and then we can see if we find (or have requests for) improvements down the road.